SECURITY.md

```diff
@@ -1,34 +1,93 @@
+Please report any **high-severity** security vulnerability, suspected or confirmed, to the Technical Steering Committee [powsybl-tsc@lists.lfenergy.org](mailto:powsybl-tsc@lists.lfenergy.org).
 
 In your e-mail, please give basic information about who you are (name and company), detailed steps to reproduce the vulnerabilities (some Python or Java code, screenshots) and the effects.
 
-For moderate or low severity security vulnerabilities, you can use public Github issues. 
+For **moderate or low-severity** security vulnerabilities, you can use public Github issues. 
 
-Github automatically reports vulnerabilities in opening pull requests. The fixes are usually available for the next version, but could lead to a patch depending on the severity.
+After reporting a potential vulnerability, you will be contacted by a PowSyBl team member within 7 calendar days. If the report proves to be a vulnerability, the library users will be informed and the vulnerability will be patched within 30 calendar days for high-severity vulnerability or in the next release train for moderate to low-severity security vulnerabilities.
 
 ## When to report a vulnerability
 
 - When you think that PowSyBl has a potential security vulnerability.
 - When you suspect a potential vulnerability but you are not sure if or how it would impact PowSyBl.
 - When you know or suspect a vulnerability on another project that is used by PowSyBl.
 
-## Security on new code
+## Supported versions
+
+The available compatible releases are listed in [powsybl-dependencies](https://github.com/powsybl/powsybl-dependencies). In case of vulnerability fix, the main branches are first fixed. 
+Only the most recent version will include the patch (through the usual release train or through a patch released outside the release train, depending on the severity).
+
+## PowSyBl security requirements
+
+### Be harmless on user environment
+
+The PowSyBl code is executed on users environments. Using the PowSyBl code must not cause damage to the users environments.
+
+### Be safe with user data
+The PowSyBl code is executed on users data. Using the PowSyBl code must not compromise the data.
+
+## Threat model
+
+### Take advantage of known vulnerabilities / bugs / bad design principles
+
+An attacker interacting with user systems powered by PowSyBl may try to use known vulnerabilities from external dependencies or take advantage of bugs or bad design principles.
+
+### Making a malicious Pull Request
+
+An attacker may try to push a malicious pull request, deliberatly introducing security vulnerabilities.
```

Comment:
You can add here that a PR can only be merged by an elected and known committer, main branches are protected.

Reply:
I think it is written below.